links/src/main/java/juja/microservices/links/controller/LinksController.java

```diff
@@ -1,20 +1,45 @@
 package juja.microservices.links.controller;
 
+import juja.microservices.links.exceptions.ApiErrorMessage;
+import juja.microservices.links.model.Link;
 import juja.microservices.links.model.SaveLinkRequest;
+import juja.microservices.links.service.LinksService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.RestController;
 
+import javax.validation.Valid;
+
+@Slf4j
 @RestController
+@ResponseBody
```

Comment:
Why do you need this annotation?
You use @RestController  - this is same

Reply:
removed